bindings/core/src/method/secret_manager.rs

```diff
@@ -47,6 +50,18 @@ pub enum SecretManagerMethod {
         /// Chain to sign the message with
         chain: Vec<u32>,
     },
+    /// Verify an ed25519 signature with a message.
+    VerifyEd25519Signature {
```

Comment:
We have `VerifyEd25519Signature` already in utils since it doesn't require the secret manager

Reply:
I don't mind moving it to utils. I just don't really know what the use-case is for the other version, and whether we should support both.

Reply:
Ah yeah without the address it's more flexible, the public key can be hashed independent
I would move it to utils and remove the old one